portal/src/main/java/com/liferay/faces/portal/render/internal/PortalTagRenderer.java

```diff
@@ -99,15 +96,6 @@ public void encodeBegin(FacesContext facesContext, UIComponent uiComponent) thro
 			// Encode the output of the JSP tag up until the point at which children should be inserted.
 			ResponseWriter responseWriter = facesContext.getResponseWriter();
 			responseWriter.write(preChildMarkup);
-
-			// Ensure that scripts are rendered at the bottom of the page.
```

Comment:
Liferay will automatically relocate scripts during non-Ajax requests. During Ajax requests, scripts do not need to be relocated since jsf.js will strip them and run them last.